programs/server/Server.cpp

```diff
@@ -399,6 +399,21 @@ int Server::main(const std::vector<std::string> & /*args*/)
     UseSSL use_ssl;
 
     MainThreadStatus::getInstance();
+    ThreadPoolInitializer::initializeAttachToThreadGroupDecorator([](PoolJob job)
```

Comment:
Does this complexity worth it? Maybe simply move this code into ThreadPool.cpp?

Reply:
@azat actually this complexity does not worth it, but we have huger linker errors because our Common uses Interpreter, check where methods of CurrentThread::attachTo is implemented.
This pull request is just to show @alexey-milovidov possible approach.